sdk/dapp-kit/src/constants/walletDefaults.ts

```diff
@@ -14,7 +14,7 @@ export const DEFAULT_STORAGE =
 export const DEFAULT_STORAGE_KEY = 'sui-dapp-kit:wallet-connection-info';
 
 export const DEFAULT_REQUIRED_FEATURES: (keyof WalletWithRequiredFeatures['features'])[] = [
-	'sui:signTransactionBlock',
+	'sui:signTransactionBlock:v2',
```

Comment:
We probably can't make this change because it will filter out all wallets that haven't migrated, right?

Reply:
We automatically add the v2 method through wallet-standard so this works, if we change how we do the fallback, this might need to be changed though